Marlin/src/module/temperature.cpp

```diff
@@ -724,12 +724,12 @@ volatile bool Temperature::raw_temps_ready = false;
     TERN_(HAS_FAN_LOGIC, fan_update_ms = next_temp_ms + fan_update_interval_ms);
 
     TERN_(EXTENSIBLE_UI, ExtUI::onPidTuning(ExtUI::result_t::PID_STARTED));
-    TERN_(DWIN_LCD_PROUI, dwinPidTuning(isbed ? PIDTEMPBED_START : PIDTEMP_START));
+    TERN_(PROUI_PID_TUNE, dwinPidTuning(isbed ? PIDTEMPBED_START : PIDTEMP_START));
```

Comment:
are you sure to revert this?

because the code is encompassed within `HAS_PID_HEATING`.

having `PROUI_PID_TUNE` is a bit redundant, no?

Reply:
if you look at say `TERN_(DWIN_LCD_PROUI, dwinMPCTuning(MPC_TEMP_ERROR));` (line 1147), this is within `MPC_AUTOTUNE`.

im going to remove `PROUI_PID_TUNE` because it simply isn't needed. if you look at **Conditionals_post.h**   
```y
#if HAS_TEMP_CHAMBER && HAS_HEATER_CHAMBER
...
#else
  #undef PIDTEMPCHAMBER
#endif

// PID heating
#if ANY(PIDTEMP, PIDTEMPBED, PIDTEMPCHAMBER)
  #define HAS_PID_HEATING 1
#endif
```

`PIDTEMPCHAMBER` gets undefined anyway, plus there are all kinds of guards when either `PIDTEMP` or `PIDTEMPBED` after